public/core.d.ts

```diff
@@ -8808,48 +8808,942 @@ declare namespace JMap {
+     * ```
+     */
     function openUpdateDialogForLayer(layerId: JId, elements: JFormElement[]): Promise<JFormMetaData[]>
+
+    /**
+     * ***JMap.Form.openCreationDialogForSubForm***
+     * 
+     * Opens the sub form creation dialog.
+     * 
+     * @param layerId the JMap layer id
+     * @param geometry the feature geometry
+     * @throws if no form dialog currently opened, or sub form not found, or no sub form table field in currently displayed form, or invalid parameters
+     * @example ```ts
+     * 
+     *  // opens the form creation dialog for layer id=3
+     *  JMap.Form
+     *    .openCreationDialogForSubForm(
+     *      4,
+     *      [
+     *        id: 3,
+     *        attributeValueByName: {
+     *          jmap_id: 3,
+     *          color: "red",
+     *          type: "tree"
+     *        }
+     *      ]
+     *    )
+     *    .then(formMetadata => console.log("Creation sub form dialog opened", formMetadata))
+     *    .catch(error => console.error(error))
+     * ```
+     */
     function openCreationDialogForSubForm(subFormId: JId, selectedParentElements: JFormElement[]): JFormMetaData
+
+    /**
+     * ***JMap.Form.openCreationDialogForSubForm***
+     * 
+     * Opens the sub form creation dialog.
+     * 
+     * @param layerId the JMap layer id
+     * @param geometry the feature geometry
+     * @throws if no form dialog currently opened, or sub form not found, or no sub form table field in currently displayed form, or invalid parameters
+     * @example ```ts
+     * 
+     *  // opens the form creation dialog for layer id=3
+     *  JMap.Form
+     *    .openCreationDialogForSubForm(
+     *      4,
+     *      [
+     *        id: 4,
+     *        parentId: 3,
+     *        elementId: 3,
+     *        parentAttributeValueByName: {
+     *          jmap_id: 3,
+     *          color: "red",
+     *          type: "tree"
+     *        },
+     *        attributeValueByName: {
+     *          id: 4,
+     *          name: "bee",
+     *          type: "nest"
+     *        }
+     *      ]
+     *    )
+     *    .then(formMetadata => console.log("Update sub form dialog opened", formMetadata))
+     *    .catch(error => console.error(error))
+     * ```
+     */
     function openUpdateDialogForSubForm(subFormId: JId, subFormElements: JFormElement[]): JFormMetaData
+
+    /**
+     * ***JMap.Form.closeCurrentDisplayedDialog***
+     * 
+     * Closes the currently displayed form.
+     * 
+     * Do nothing if no form displayed.
+     * 
+     * @example ```ts
+     * 
+     *  // closes currently displayed form
+     *  JMap.Form.closeCurrentDisplayedDialog()
+     * ```
+     */
     function closeCurrentDisplayedDialog(): void
+
+    /**
+     * ***JMap.Form.getFormValues***
+     * 
+     * Returns default values if form has not been edited, else form values.
+     * 
+     * This is a technical method used by JMap App, you should never used this function
```

Comment:
you should never used this function => you should never have to use this function ou you should never use this function